CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pip.
 
 ## [0.0.x](https://github.com/con/tributors/tree/master) (0.0.x)
+ - add ability to run twice to action (0.0.21)
```

Comment:
```suggestion
 - add to action ability to run command twice (0.0.21)
```
reads a bit better to me.

BUT IMHO it is more of a code issue/bug -- I would expect that command would just do everything necessary on a single run.  It is like if `git commit` needed two runs to commit, or `rm` two runs to remove and then we really wanted an explicit option (disabled by default) to really make it happen.  Or what am I missing, in that why it might be desired to run multiple times to achieve the mission?

Reply:
Yeah I think so too, this was just the easier fix. The problem is that we have two things that happen - updating a tributors file (e.g., finding a new contributor) and then reading an existing one and updating with metadata. The bug is that given the first happens (and for a different means I think) we don't do the second at the same time, at least without the second run, where the main action is for the metadata. My proposal is to allow this bit of a messy fix for now until I (or someone else) can make some time to work on it. I'm not entirely happy with this library, TBH, and it's feeling about the right time to refactor and fix up some things. If you want to spec out a new proposal for a design (Google doc, markdown, issue, whatever) that would be fun! I'm working on like three exciting side projects now but I'll add this to my queue for when those run out, lol.

Reply:
And I'd appreciate some help to think about the redesign - if you want to open an issue and start putting down everything terrible about the current design (or ideas for a version II) I'd hugely appreciate (and enjoy!) that too.

Reply:
redesign -- a nice idea, will see if I get some time to contribute.

two runs -- ok, let's go with the workaround.  But may be let's make double run by default? this way there would be a better chance it would operate correctly by default, and allow to revert to single run for odd cases?